internal/compiler/utils.go

```diff
@@ -16,12 +19,35 @@ const (
 	AuthorizationPolicySchema SchemaFile = "authorizationPolicy.json"
 )
 
-var schemaDefinitions = map[SchemaFile]interface{}{}
+var schemaDefinitions = map[SchemaFile]any{}
+
+var loadOnce = sync.OnceValue(func() error {
```

Comment:
This just moves initialization from `init()` to when it's actually needed. This isn't so much about performance, but just that there's no need to init authoziation policy in `opa eval`, `check`, etc